### DISCUSSION TITLE
Plugin Suggestion Process

### DISCUSSION THREAD

**Discussion by artf** [GrapesJS/grapesjs#5985] [Plugin Suggestions]
Hi folks, welcome to the GrapesJS Plugin Suggestions discussion. Here, you can suggest new plugins you'd like to see in GrapesJS.

## The Process
Similar to the [Roadmap Process](https://github.com/GrapesJS/grapesjs/discussions/5291) we will also follow here two fundamental steps, the first one is the [Plugin Suggestions](https://github.com/GrapesJS/grapesjs/discussions/categories/plugin-suggestions) discussion category, which will be used to collect and vote for new suggestions from the community, and then there is the [Roadmap Project](https://github.com/orgs/GrapesJS/projects/1) which will provide the status of the accepted suggestion.

## From Suggestion to Release
1. [Open a new discussion in the Plugin Suggestions category](https://github.com/GrapesJS/grapesjs/discussions/new?category=plugin-suggestions).
   * Provide a detailed description of the plugin.
       eg. dependencies, options, etc.
1. Now people can **comment and vote** on the suggestion before bringing it into the Roadmap.
   * Use the default voting mechanism provided by GitHub Discussions.
     <img width="247" alt="Screenshot 2023-08-09 at 20 54 20" src="https://github.com/GrapesJS/grapesjs/assets/11614725/f78f32f6-acff-412e-b7ce-e2ab9695c90a">

1. The suggestion should **reach at least 10 votes** before being evaluated and accepted for the Roadmap.
1. Once the suggestion reached the minimal amount of votes, it will be **evaluated and moved into the Roadmap**.
1. The suggestion can now start its **development cycle**.
    * **Accepted**: here we can discuss all the necessary information in order to start working on the suggestion.
    * **In progress**: the suggestion is in development.
    * **In review**: the suggestion can be published as a release candidate for review.
    * **Done**: the suggestion is now officially published.

If you have any questions/suggestions about the process, you can post them directly here in this discussion post.

Looking forward to your amazing plugin ideas!

**Comment by RakulAgn** [comment]
Looking for a Proper Custom Rich Text Editor Recently Ckeditor 4 is Completely Deprecated

**Reply by artf** [reply]
ok sure, but what about following the mentioned process?! 😅

**Reply by RakulAgn** [reply]
Sure Let me create a Detailed information in discussion
